src/pybind/mgr/dashboard/services/nvmeof_cli.py

```diff
@@ -51,6 +51,45 @@ def remove_nvmeof_gateway(_, name: str, daemon_name: str = ''):
         return -errno.EINVAL, '', str(ex)
 
 
+B = "B"
+K, KB = "K", "KB"
+M, MB = "M", "MB"
+G, GB = "G", "GB"
+T, TB = "T", "TB"
+P, PB = "P", "PB"
+UNITS = {
+    B: 1,
+    KB: 1024,
+    K: 1024,
+    MB: 1024**2,
+    M: 1024**2,
+    GB: 1024**3,
+    G: 1024**3,
+    TB: 1024**4,
+    T: 1024**4,
+    PB: 1024**5,
+    P: 1024**5
+}
```

Comment:
I ❤️ Python dict/list comprehensions:
```suggestion
UNITS = { unit: 1024**mult for mult, unit in enumerate(['B', 'K', 'M', 'G', 'P'])}
>>> {'B': 1, 'K': 1024, 'M': 1048576, 'G': 1073741824, 'P': 1099511627776}
```

Reply:
haha it is indeed more elegant! but since we need the dict to contain both "K" and "KB" (for current cli compatibility), I am not sure how to elegantly implement this - I can do 2 list comprehensions  - one for the single letters notation and another for the 2 letters notation and then merge the dicts but I am not sure if I will end up causing more obfuscation than elegance. do you have an idea?

Reply:
Now with both k, KB and KiB support (although technically k should be x1000 multiples, and ki x1024's):
```py
MULTIPLES = ['', 'K', 'M', 'G', 'T', 'P']
{
	f"{prefix}{suffix}": 1024**mult
	for mult, prefix in enumerate(MULTIPLES)
	for suffix in ['', 'B', 'iB']
	if not (prefix == '' and suffix == 'iB')
}

>>> {'': 1, 'B': 1, 'K': 1024, 'KB': 1024, 'KiB': 1024, 'M': 1048576, 'MB': 1048576,
'MiB': 1048576, 'G': 1073741824, 'GB': 1073741824, 'GiB': 1073741824,
'T': 1099511627776, 'TB': 1099511627776, 'TiB': 1099511627776, 
'P': 1125899906842624, 'PB': 1125899906842624, 'PiB': 1125899906842624}
```